pkg/components/images.go

```diff
@@ -16,7 +16,7 @@ package components
 // Default registries for Calico and Tigera.
 const (
 	CalicoRegistry = "docker.io/"
-	TigeraRegistry = "gcr.io/unique-caldron-775/cnx/"
+	TigeraRegistry = "quay.io/"
```

Comment:
Just wanted to double check this, it's intentional, right?

Reply:
The comment on the line below suggests that it could be, although I'd want confirmation.

Reply:
@Brian-McM @matthewdupre Yes, this is intentional. TigeraRegistry is where Operator will expect published images to come from; before we release a minor version, we expect any images we generate to come from our dev repo, since obviously we aren't publishing them, but after release if we don't change this then customers' operators will attempt to pull from gcr.io.

(In actuality this won't happen, because it's always caught by our post-release validations, but the issue is the same)

You can see the instructions referenced [in the release docs](https://github.com/tigera/operator/blob/release-v1.32/RELEASING.md#updating-versions), which IIRC we had to add the first time we did a release without updating this field.